docs/source/getting_started/getting_started.ipynb

```diff
@@ -45,19 +46,151 @@
+    "\n",
+    "#### MAAP Jupyter Sidebar\n",
+    "![Jupyter Sidebar](_static/interface2.png)\n",
+    "\n",
+    " - File Browser\n",
+    " - Running Terminals & Kernels\n",
+    " - Git Repo Interface (if this folder is a Git repo)\n",
+    " - Jupyter Command Search\n",
+    " - Job listing\n",
+    " - Property Inspector\n",
+    " - Open Tabs\n",
+    " - Extension manager\n",
+    "\n",
+    "#### MAAP Blue Sidebar\n",
+    "\n",
+    " - Workspaces: See workspaces, share them, as well as configure settings\n",
+    " - Stacks: See available platforms for workspaces & required memory\n",
+    " - Administration: Control the configuration & policies for your installation.\n",
+    " - Organizations: allow groups of developers to collaborate with private & shared workspaces. Resources & permissions are controlled & allocated within the organization by admin.\n",
+    " - Profile (bottom, labeled with your name): See account info, logout\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### MAAP Storage Options\n",
+    "![Jupyter annotated folders](_static/storageoptions.png)\n",
+    "\n",
+    "#### My root folder (\"in\" the workspace)\n",
+    " - The Jupyter root directory is mounted to `/projects`.  Stuff in here persists across sessions.\n",
+    "     - Use this for code-related items, smaller data storage\n",
+    "     - Git is more likely to behave predictably here compared to other storage\n",
+    "     - This is also the place to make persistent conda environments (covered in another section), but make sure to not make a conda env inside a git-tracked folder, or if you do add it to the .gitignore. If git is tracking an env, it could cause your workspace to crash. \n",
+    " - Uses local (not network-based) file system; generally faster and more reliable for “normal” file operations, but expensive\n",
+    "\n",
+    "#### My Private Bucket\n",
+    "`my-private-bucket` is an S3 bucket with persistent storage, but accessible only to you and others in the workspace\n",
+    " \n",
+    " - Use for large data storage\n",
+    "\n",
+    "#### My Public Bucket\n",
+    "`my-public-bucket` is an S3 bucket with persistent storage. It is the same as `shared-buckets/<username>/` — anything you put in here will be accessible to other users via `shared-buckets/<your username>`.\n",
+    "\n",
+    " - Use for large data storage for files that you want to share across workspaces\n",
+    "\n",
+    "#### Mounting your MAAP workspace on your local computer\n",
+    "If you prefer to work on your local computer, or to drag-and-drop copy files from your computer to/from MAAP, you access the workspace via SSH. The process for doing this is below."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Accessing MAAP workspaces over SSH\n",
```

Comment:
I followed the add SSH steps in OPS and when I try to rename ssh to .ssh, I get an error that .ssh already exists. I press overwrite and I get a 409 error not 404. When I list the contents of .ssh, it only has known hosts, not the authorized_keys file that I just created. I tried this a second time with the same result. If I `ls -al`, I can see both ssh folders listed as like .ssh.15f4974f-888c-4397-9ee9-7cd47f32c346 with my authorized_keys file inside. 
If I try deleting the existing .ssh and repeating the steps, I get a 404 and I see my authorized_keys in .ssh
When I click “Display SSH Info”, I see `ssh root@undefined -p undefined`